ports/raspberrypi/bindings/tca9555r/__init__.c

```diff
@@ -0,0 +1,743 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2022 Jeff Epler for Adafruit Industries
+ * Copyright (c) 2016 Scott Shawcroft
```

Comment:
Likely the copyright data should be updated

Reply:
Ah yes. What should I update that to. I assumed all CircuitPython code would be copyright Adafruit?

Reply:
Nope, it should be your copyright. It should reflect who wrote the non-trivial code.